apps/emqx_ft/src/emqx_ft_api.erl

```diff
@@ -57,29 +59,131 @@ schema("/file_transfer/files") ->
             tags => [<<"file_transfer">>],
             summary => <<"List all uploaded files">>,
             description => ?DESC("file_list"),
+            parameters => [
+                ref(following),
+                ref(emqx_dashboard_swagger, limit)
+            ],
             responses => #{
                 200 => <<"Operation success">>,
+                400 => emqx_dashboard_swagger:error_codes(
+                    ['BAD_REQUEST'], <<"Invalid cursor">>
+                ),
                 503 => emqx_dashboard_swagger:error_codes(
-                    ['SERVICE_UNAVAILABLE'], <<"Service unavailable">>
+                    ['SERVICE_UNAVAILABLE'], error_desc('SERVICE_UNAVAILABLE')
+                )
+            }
+        }
+    };
+schema("/file_transfer/files/:clientid/:fileid") ->
```

Comment:
Nit: the API method design is driven by the edge case: that we have multiple files exported with the same fileid.
But currently, I have no ideas how to improve either 😄

Reply:
Heh. 🙃 Well, as long as it stands for _collection of files, scoped by client id / file id_ I think that should be fine (where REST is concerned).